docker-compose.yml

```diff
@@ -40,40 +38,45 @@ services:
       timeout: 1s
       retries: 3
       start_period: 30s   # Failures in 30 seconds do not mark container as unhealthy
-    networks:
-      datastore-network:
     depends_on:
       db:
         condition: service_healthy
 
   client:
+    profiles: ["client"]
```

Comment:
In case I would like to build all of the images, or remove all of the containers. Do you know if docker supports a command which enables everything inside the docker compose when using --profiles?

Reply:
This works for building:
 `docker compose --profile client --profile loader --profile integration build`
 
 As I am only using the profile to ensure some containers are not initially started, I could put them all in the same profile named `do-not-start`.
 
 I think `docker compose down` always gets them all.